src/services/contentAggregator.js

```diff
@@ -52,8 +76,14 @@ export async function addContextToContent(dataPromise, ...dataArgs)
     ...(addLikeCount && ids.length === 1 ? { likeCount: await fetchLikeCount(item.id) } : {}),
     ...(addResumeTimeSeconds ? { resumeTime: resumeTimeData?.[item.id] } : {}),
     ...(addLastInteractedChild ? { lastInteractedChild: lastInteractedChildData?.[item.id] } : {}),
+    ...(addNextLesson ? { nextLesson: nextLessonData?.[item.id] } : {}),
   })
-  
+
+  if (lastInteractedParent) {
```

Comment:
```suggestion
  if (addLastInteractedParent) {
```
to match param name format